examples/search.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+require  dirname(__FILE__) . '/../src/sajari.php';
+
+use Sajari\WithEndpoint;
+use Sajari\Client;
+use Sajari\Request;
+
+$opts = [new WithEndpoint('server_address')];
+
+$c = new Client('myproject', 'mycollection', $opts);
+
+$r = new Request();
+$r->setBody("request body of words to search for");
+$r->setMaxResults(10);
+$r->setPage(1);
+$r->setFilter(
+    Sajari\All([
+        Sajari\EqualTo("type", "webpage"),
+        Sajari\EqualTo("category", "education"),
```

Comment:
@tbillington filters look great. Maybe can put in a `pageviews > x` and a `not` just to show another operator or two?